draft-ietf-tls-tls13.md

```diff
@@ -2406,20 +2406,28 @@ isn't forcing the local system into a small subgroup.
 ECDHE parameters for both clients and servers are encoded in the
 the opaque key_exchange field of a KeyShareEntry in a KeyShare structure.
 
-For secp256r1, secp384r1 and secp521r1, the contents are the byte string
-representation of an elliptic curve public value following the conversion
-routine in Section 4.3.6 of ANSI X9.62 {{X962}}.
-
-Although X9.62 supports multiple point formats, any given curve
-MUST specify only a single point format. All curves currently
-specified in this document MUST only be used with the uncompressed
-point format (the format for all ECDH functions is considered
-uncompressed). Peers MUST validate each other's public value Y by ensuring
-that the point is a valid point on the elliptic
-curve.
-
-For the curves secp256r1, secp384r1 and secp521r1, the appropriate
-validation procedures are defined in Section 4.3.7 of {{X962}}
+For secp256r1, secp384r1 and secp521r1, the contents are the serialized
+value of the following struct:
+
+~~~~
+       struct {
+           uint8                legacy_form = 4;
+           opaque               X[coordinate_length];
+           opaque               Y[coordinate_length];
+        } UncompressedPointRepresentation;
```

Comment:
This probably doesn't need to be indented so much (~~~ causes a three-space indent).